lib/utils.js

```diff
@@ -421,7 +421,7 @@ exports.sliceArgs = sliceArgs;
 
 function now(dialect) {
   const now = new Date();
-  if (['postgres', 'sqlite'].indexOf(dialect) === -1) {
+  if (['mysql','postgres', 'sqlite'].indexOf(dialect) === -1) {
```

Comment:
No it does not trigger a warning and yes it just drops the milliseconds. I tested on 5.7 and 5.6.